tests/sentry/notifications/utils/test_participants.py

```diff
@@ -521,3 +523,66 @@ def test_get_owner_reason_assigned(self):
                 event=event,
             )
             assert owner_reason is None
+
+
+@apply_feature_flag_on_cls("organizations:issue-alert-fallback-targeting")
+class GetSendToFallthroughTest(TestCase):
+    def get_send_to_fallthrough(
+        self, event: Event, fallthrough_choice: Optional[FallthroughChoiceType] = None
+    ) -> Mapping[ExternalProviders, Iterable[Union["Team", "User"]]]:
+        return get_send_to(
+            self.project,
+            target_type=ActionTargetType.ISSUE_OWNERS,
+            target_identifier=None,
+            event=event,
+            fallthrough_choice=fallthrough_choice,
+        )
+
+    def store_event(self, filename: str) -> Event:
+        return super().store_event(data=make_event_data(filename), project_id=self.project.id)
+
+    def setUp(self):
+        self.user2 = self.create_user(email="baz@example.com", is_active=True)
+        self.user3 = self.create_user(email="bar@example.com", is_active=True)
+
+        self.team2 = self.create_team(
+            organization=self.organization, members=[self.user, self.user2]
+        )
+        self.project.add_team(self.team2)
+
+        ProjectOwnership.objects.create(
+            project_id=self.project.id,
+            schema=dump_schema(
+                [
+                    grammar.Rule(Matcher("path", "*.py"), [Owner("team", self.team2.slug)]),
+                    grammar.Rule(Matcher("path", "*.lol"), []),
+                ]
+            ),
+            fallthrough=False,  # test with fallthrough off to ensure the new fallthrough logic is used
+        )
+
+        # turn off slack for teams
+        for user in [self.user, self.user2]:
+            NotificationSetting.objects.update_settings(
+                ExternalProviders.SLACK,
+                NotificationSettingTypes.ISSUE_ALERTS,
+                NotificationSettingOptionValues.NEVER,
+                user=user,
+            )
+
+    def test_no_fallthrough(self):
+        event = self.store_event("empty.lol")
+        assert self.get_send_to_fallthrough(event, None) == {}
+
+    def test_fallthrough_no_one(self):
+        event = self.store_event("empty.lol")
+        assert self.get_send_to_fallthrough(event, FallthroughChoiceType.NO_ONE) == {}
+
+    def test_fallthrough_all_members(self):
+        event = self.store_event("all.py")
```

Comment:
Do you need to write a test case for `"empty.lol"` with using `FallthroughChoiceType.ALL_MEMBERS`?